GitCommands/Git/GitGpgController.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using GitUIPluginInterfaces;
+using System.Threading.Tasks;
+
+
+namespace GitCommands.Gpg
+{
+    public enum CommitStatus
+    {
+        GoodSignature = 0,
+        SignatureError = 1,
+        MissingPublicKey = 2,
+        NoSignature = 3
+    };
+
+    public enum TagStatus
+    {
+        OneGood = 0,
+        OneBad = 1,
+        Many = 2,
+        NoPubKey = 3
```

Comment:
as @jbialobr suggested - add the default "None" or "NoTag". the likelihood of no signature is much greater than any other state

Reply:
ok.